pages/linux/at.md

```diff
@@ -0,0 +1,11 @@
+# at
+
+> Executes commands at a specified time.
+
+- Open 'at' prompt to execute commands at specific time. When you done press Ctrl+D to register job(s):
+
+`at {{hh:mm:ss}}`
+
+- Execute file (-f) and mail result(-m):
```

Comment:
I wouldn't include the -m option in this example. We have the space to include it as a separate example.

Also, what is file meant to be? An executable script? A list of commands?